lib/daru.rb

```diff
@@ -84,8 +94,8 @@ def plotting_library= lib
   gem 'spreadsheet', '~>1.1.1'
   require 'spreadsheet'
 rescue LoadError
-  STDERR.puts "\nInstall the spreadsheet gem version ~>1.1.1 for using"\
-  ' spreadsheet functions.'
+  Daru.error_stream.puts "\nInstall the spreadsheet gem version ~>1.1.1 for using"\
+  ' spreadsheet functions.' if Daru.error_stream
```

Comment:
I'd abstract this into method like `Daru.error(message)`. Then it would be easy to test.